notebooks/Query_butler_WEP_outputs.ipynb

```diff
@@ -0,0 +1,321 @@
+{
```

Comment:
<p>Can we establish some sort of template header for the notebooks we are going to add in this repo? Something like what we did in the <code>ts_analysis_notebooks</code> like this one: <a href="https://github.com/lsst-ts/ts_analysis_notebooks/blob/develop/aos/closed_loop/wepSourceSelectionWithWcs.ipynb" target="_blank">https://github.com/lsst-ts/ts_analysis_notebooks/blob/develop/aos/closed_loop/wepSourceSelectionWithWcs.ipynb</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/aos_analysis/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>